TCL4 Data Management/utm-tcl4-dmp-daa.yaml

```diff
@@ -342,7 +367,9 @@ definitions:
       expectedTimeToHover:
         description: >-
           Expected time required for a vehicle to arrest forward motion and
-          hover at a position. (sec)
+          hover at a position.
+
+          Please report in milliseconds (ms).
```

Comment:
@bmcook1  the following note is missing here
NOTE:
            DMP-Rev3 asks asks this to be reported with 3 decimals of precision.
            However, the reference here is for "seconds" which requires an integer.
            The workaround for this is to submit in milliseconds.

Reply:
This was intentionally omitted here. The DMP-Rev3 does not ask this element to be reported in 3 decimal places. The previous description is consistent (didn't ask for 3 decimal places).